spec/parity/backup_spec.rb

```diff
@@ -15,6 +17,27 @@
       with(heroku_production_to_development_passthrough)
   end
 
+  it "correctly parses database.yml files that include ERB and environment variables" do
+    database_name = "parity_erb_development"
+    ClimateControl.modify DEVELOPMENT_DATABASE_NAME: database_name do
```

Comment:
I think if I do that I have to specify the DB name inside and outside the block. I'm open to that change, but since it's only relevant in one example I'm not sure if setting up a separate context is worth it.